client/coral-plugin-commentbox/CommentBox.js

```diff
@@ -1,6 +1,7 @@
 import React, {Component, PropTypes} from 'react';
 import {I18n} from '../coral-framework';
 import translations from './translations.json';
+import {Button} from 'coral-ui';
```

Comment:
Is `coral-ui` published to npm? I thought we were trying to minimize custom UI in talk. No objections, just want to make sure that this reflects an intentional change in our earlier decisions.